commonmark-ext-autolink/src/main/java/org/commonmark/ext/autolink/internal/AutolinkPostProcessor.java

```diff
@@ -67,8 +99,12 @@ private static Text createTextNode(String literal, Span span, SourceSpan sourceS
     }
 
     private static String getDestination(LinkSpan linkSpan, String linkText) {
-        if (linkSpan.getType() == LinkType.EMAIL) {
+        LinkType type = linkSpan.getType();
+
+        if (type == LinkType.EMAIL) {
             return "mailto:" + linkText;
+        } else if (type == LinkType.WWW) {
+            return "http://" + linkText;
```

Comment:
Hmm, why `http://` and not `https://`? I feel like nowadays we should default to the latter. If anyone wants `http://`, we could add it as an another option to the extension builder.

Reply:
Hmm ok, let's copy them in that case.